<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>396</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>515</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="mainLabel">
       <property name="text">
        <string>1) Input number of failures for each UAV main system and click &quot;Enter&quot;.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mainLabel_2">
       <property name="text">
        <string>2) Click &quot;Next Page&quot; for detailed breakdown of reliability for each UAV main system.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="1">
          <widget class="QLineEdit" name="input_1"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="input_2_label">
           <property name="text">
            <string>input_2_label</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="input_2"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="input_3_label">
           <property name="text">
            <string>input_3_label</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="input_3"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="input_1_label">
           <property name="text">
            <string>input_1_label</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="input_4_label">
           <property name="text">
            <string>input_4_label</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="input_4"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="input_5_label">
           <property name="text">
            <string>input_5_label</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="input_5"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="input_6_label">
           <property name="text">
            <string>input_6_label</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="input_6"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="output_1_label">
         <property name="text">
          <string>output_1_label</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="output_1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="enter_button">
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Next Page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="error_message">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
